Casks/dosbox-x.rb

```diff
@@ -34,4 +34,8 @@
   homepage "https://dosbox-x.com/"
 
   app "dosbox-x/dosbox-x.app"
+
+  zap trash: [
+    "~/Library/Preferences/com.dosbox-x.plist",
+  ]
```

Comment:
For single trash path, the style preference is a single line:
```suggestion
  zap trash: "~/Library/Preferences/com.dosbox-x.plist"
```

Reply:
updated

Reply:
actually just found another file for zap. list updated again.